pkg/network/setup/netpod/netpod.go

```diff
@@ -564,6 +568,32 @@ func (n NetPod) lookupMasquradeBridge(desiredIfacesSpec []nmstate.Interface) *nm
 	return nil
 }
 
+func calcQueuesCountByIfaceName(desiredQueueCount int,
+	ifaces []v1.Interface,
+	ifaceStatuses []v1.VirtualMachineInstanceNetworkInterface) map[string]int {
+
+	ifaceStatusesInDomainByName := vmispec.IndexInterfaceStatusByName(ifaceStatuses,
+		func(ifaceStatus v1.VirtualMachineInstanceNetworkInterface) bool {
+			return vmispec.ContainsInfoSource(ifaceStatus.InfoSource, vmispec.InfoSourceDomain)
+		})
```

Comment:
This filter in IndexInterfaceStatusByName is nice, but it burns some brain cycles for no reason...
I think if we want to have this, we need to keep it readable.
 
how about adding a 
```
func hasInfoSourceDomain(ifaceStatus v1.VirtualMachineInstanceNetworkInterface) bool {
    return vmispec.ContainsInfoSource(ifaceStatus.InfoSource, vmispec.InfoSourceDomain)
}
```
so 
```
ifaceStatusesInDomainByName := vmispec.IndexInterfaceStatusByName(ifaceStatuses, hasInfoSourceDomain)``` we be readable ?

Reply:
Done.